lib/matplotlib/backends/backend_qt5.py

```diff
@@ -17,10 +18,19 @@
 from matplotlib.backends.qt_editor._formsubplottool import UiSubplotTool
 from . import qt_compat
 from .qt_compat import (
-    QtCore, QtGui, QtWidgets, __version__, QT_API,
-    _devicePixelRatioF, _isdeleted, _setDevicePixelRatioF,
+    QtCore,
+    QtGui,
+    QtWidgets,
+    _getSaveFileName,
+    __version__,
+    QT_API,
+    _devicePixelRatioF,
+    _isdeleted,
+    _setDevicePixelRatioF,
 )
 
+from PyQt5 import QtNetwork
```

Comment:
Have you tried this with PySide? I guess this needs to go in `qt_compat` as well.

Reply:
Yes, it certainly should. I have modified the qt_compat and have tested with PySide2, it works OK

Reply:
We can't have pyqt5 imports in this file, all of that needs to happen in `qt_compat` and shimmed across pyqt4 / pyside / pyqt5 / pyside2